controllers/grid/users/author/AuthorGridHandler.inc.php

```diff
@@ -234,10 +234,10 @@ public function setDataElementSequence($request, $rowId, $gridDataElement, $newS
         if (!$this->canAdminister($request->getUser())) {
             return;
         }
-        $authorDao = DAORegistry::getDAO('AuthorDAO'); /** @var AuthorDAO $authorDao */
-        $author = $authorDao->getById($rowId);
+
+        $author = Repo::author()->get($rowId);
```

Comment:
It may be a good idea to cast the `$rowId` to an `int`, like `Repo::author()->get((int) $rowId);`. If the tests pass, this instance is probably ok, but whenever working with the old UI handlers, I find it's a little dangerous to assume a variable's type.

Reply:
Isn't that `int` already, because of the `public function setDataElementSequence` signature (declaring `* @param $rowId int`) of the `GridHandler.inc.php`?

Reply:
A docblock won't be strictly enforced. So even though it says `* @param $rowId int` someone could pass a string.